InitNiKom/InitNiKom.c

```diff
@@ -28,7 +28,7 @@ void createDirOrDie(char *dirname) {
   BPTR lock;
   struct FileInfoBlock __aligned fib;
   
-  if(lock = Lock(dirname, ACCESS_READ)) {
+  if((lock = Lock(dirname, ACCESS_READ))) {
```

Comment:
Well, I guess it's possible to setup something like https://travis-ci.org/ to automatically make test builds using gcc when new code is pushed.